fastchat/conversation.py

```diff
@@ -223,6 +224,17 @@ def get_prompt(self) -> str:
                 else:
                     ret += role + ":"
 
+            return ret
+        elif self.sep_style == SeparatorStyle.COHERE:
+            ret = ""
+            if self.system_message:
+                ret += system_prompt + self.sep
+            for role, message in self.messages:
+                if message:
+                    ret += role + ": " + message + self.sep
+                else:
+                    ret += role + ":"
+
```

Comment:
I dont think we need this? as we interact with cohere through its SDK which only requires chat history.

Reply:
We will likely need this in the near future, so consider this future-proofing which we can remove if it turns out to be obsolete